docs/mkdocs.yaml

```
@@ -0,0 +1,71 @@
+site_name: mechaphlowers
+site_url : http://mechaphlowers.readthedocs.com
+repo_url: https://github.com/phlowers/mechaphlowers
+
+theme:
+  name: "material"
+  logo: _static/logos/mecaphlowers_logotxt-white1.png
+  favicon: _static/mecaphlowers_logotxt-white1.png
```

The favicon doesn't show on my PC - I think this is the correct path?
```suggestion
  favicon: _static/logos/mecaphlowers_logotxt-white1.png
```